bash_aliases.sh

```diff
@@ -1,26 +1,36 @@
-
+#!/bin/sh
 # Venv ALiases                                                                   
-alias venv-list='locate --regex "bin/activate"$'        # Locate /bin/activate paths for both conda and venv                         
-alias seder='sed  's/bin\/activate$//' '  # Cannot remember
+alias venv-list='locate --regex "bin/activate"$'    # Locate /bin/activate paths for both conda and venv                         
+alias seder='sed  's/bin\/activate$//' '            # Cannot remember
 
 #alias python=python2.7 
 
 # Conda Aliases                                                                  
-alias conda-remove-env='conda env remove --name $1'                             
-alias conda-create-env='conda create --name $1'                                 
-alias conda-lsp='conda list --name $1' # List conda packages                                          
-alias conda-ls='conda env list'        # List conda environments                                                  
-                                                                                 
-#Git Aliases                                                                     
-alias git-graph='git log --graph --oneline'                      # A simpler heuristic                                   
-alias git-show-lastn=' bash /home/meow/scripts/gitshowcom.sh $1' # Look behind at the nth commit back in the log.
+alias conda-create-env='conda env create --name'    # Creates a conda enviroment with the name as first argument($1)
```

Comment:
was $1 redundant?

Reply:
Yes, you can implement the $1 in your ```gitshowcom.sh``` script rather than using it in an alias :)

Reply:
Is there any convention to not use arguments within aliases?